newrelic_lambda_cli/integrations.py

```diff
@@ -517,12 +568,12 @@ def install_log_ingestion(
     Returns True for success and False for failure.
     """
     assert isinstance(input, IntegrationInstall)
-    function = get_function(input.session, "newrelic-log-ingestion")
+    function = get_newrelic_log_ingestion_function(input.session)
```

Comment:
Since the log ingestion function will only exist if the stack exists we can just check the stack status and remove this function check. This was here before because previously we could only install one log ingestion function per region but now we can install multiple per region so this isn't needed. If the old log ingestion function exists this should be fine as it is now unused though we may want to notify the user that they should remove it. I've made some suggested edits that remove this function if-else check but after that you'll have to tab in the remaining inner conditional block that checks the stack status.

Reply:
Yes I have tested by installing the CLI locally and using it for integrating the lambda and adding subscription.